### PR TITLE
latest Zanata translation changes in kie-wb-distributions from 2016-09-29

### PR DESCRIPTION
**Pull request by kiereleaseuser**

### CODE REVIEW

File: kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties [hunk 1]
```diff
@@ -14,3 +14,5 @@ loginAsAnotherUser=Connectez-vous en tant qu'un autre utilisateur
 logoutSuccssful=D\u00e9connexion r\u00e9ussie
 # translation auto-copied from project KIE webapp distributions, version 6.2.0, document org.kie/kie-wb-webapp/org/kie/workbench/client/resources/i18n/LoginConstants, author croe@redhat.com
 loginAgain=Connectez-vous \u00e0 nouveau
+loadingPleaseWait=Veuillez patienter
+loadingApplication=Chargement de l'application...
```
Comment (manstis) [inline kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties]: This hasn't been escaped... did the replacer plugin run?

Comment (mbiarnes) [inline kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties]: yepp - I will test it locally

Comment (manstis) [inline kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties]: @mbiarnes Hi.. it looks like ```LoginConstants_fr.properties``` will be ignored by https://github.com/jboss-integration/droolsjbpm-build-bootstrap/blob/master/pom.xml#L744

**Comment by manstis** [comment]
This build *may* succeed as Jenkins does not compile the "full" profile. However this PR should not be merged as it stands as it contains an non-escaped character in ```LoginConstants_fr.properties```.. Ideally we should compile ```kie-wb-distributions``` with the "full" profile.. but that'd make a huge impact on our CI cycle (for all PRs).

File: kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties (hunk 1 above)
Comment (mbiarnes) [inline kie-wb/kie-wb-webapp/src/main/resources/org/kie/workbench/client/resources/i18n/LoginConstants_fr.properties]: reviewing the script, for kie-wb.distribution it does:
mvn -B zanata:pull-module
mvn replacer:replace -N
mvn native2ascii:native2ascii followed but
mvn -B zanata:pull-module -Dproductized
mvn replacer:replace -N -Dproductized
mvn native2ascii:native2ascii -Dproductized

I can't see something wrong there.

**Comment by mbiarnes** [comment]
this PR will be closed as it has not escaped characters

**Pull request closed without merging**